src/wallet/wallet.cpp

```diff
@@ -4969,10 +4969,10 @@ CAmount CWalletTx::GetAvailableCredit(const std::optional<int>& asOfHeight, bool
     CAmount* cache = nullptr;
     bool* cache_used = nullptr;
 
-    if (filter == ISMINE_SPENDABLE) {
+    if ((filter & ISMINE_SPENDABLE) != ISMINE_NO) {
```

Comment:
I think it is slightly clearer to just use:
```suggestion
    if (filter & ISMINE_SPENDABLE) {
```
Non-blocking.